pkgpanda/util.py

```diff
@@ -150,6 +151,38 @@ def get_requests_retry_session(max_retries=4, backoff_factor=1, status_forcelist
     return session
 
 
+def stream_remote_file_with_retries(out_filename, url, retries=4):
+    def cleanup():
+        try:
+            os.remove(out_filename)
+        except Exception:
```

Comment:
Can you perhaps be more particular with the exceptions that you wish to ignore?

Reply:
This is also taken from here: https://github.com/dcos/dcos/blob/2897e2459e460033fb2b72099d4aac29e2b58937/pkgpanda/util.py#L185-L189